packages/node-cli/src/validation/InteractiveAnswersValidator.js

```diff
@@ -181,6 +181,14 @@ module.exports = {
 		return !fileSystemService.fileExists(path.join(parentFolderPath, filenameWithExtension))
 			? VALIDATION_RESULT_SUCCESS
 			: VALIDATION_RESULT_FAILURE(NodeTranslationService.getMessage(ANSWERS_VALIDATION_MESSAGES.FILE_ALREADY_EXISTS));
+	},
+
+	validateFolderDoesNotExists(path) {
```

Comment:
Just a grammar question, shouldn't it be `validateFolderDoesNotExist` and `validateSuiteScriptFileDoesNotExist` without the final `s`?
Anyway it's ok for me.

Reply:
You are correct. I updated it.